src/istgt_iscsi.c

```diff
@@ -1965,6 +1966,28 @@ istgt_iscsi_op_login(CONN_Ptr conn, ISCSI_PDU_Ptr pdu)
 			}
 			MTX_UNLOCK(&conn->istgt->mutex);
 
+			/* check existing live connections other than current
+			 * initiator. If any of existing connection found live then
+			 * reject the current connection request
+			 */
+			ISTGT_TRACELOG(ISTGT_TRACE_ISCSI,
+				"checking existing live connections "
+				"other than from initiator=%s isid=%"PRIx64", tsih=%u, cid=%u\n",
+				conn->initiator_name, isid, tsih, cid);
+			rc = istgt_iscsi_is_existing_conn_alive(conn);
```

Comment:
better to do this only for cstor variant.
consider adding 'REPLICATION' macro

Reply:
Updated